dwi_ml/data/hdf5/hdf5_creation.py

```diff
@@ -154,10 +158,16 @@ def __init__(self, root_folder: Path, out_hdf_filename: Path,
             List of subject names for each data set.
         groups_config: dict
             Information from json file loaded as a dict.
+        dps_keys: List[str]
+            List of keys to keep in data_per_streamline. Default: None.
         step_size: float
             Step size to resample streamlines. Default: None.
+        nb_points: int
+            Number of points per streamline. Default: None.
```

Comment:
Could you add here and in the code a verification that you only use one of step_size and nb_points?